### PR TITLE
Merge rel/2.0.1 to dev

### PR DESCRIPTION
**Pull request by smitpatel**
We always merge back rel branches to dev. This has been out of sync and now there are multiple conflicts. 

If you are assigned to this PR, then please push a commit to resolve conflicts with dev. So that this branch can be merged to dev & dev becomes consistent with rel/2.0.1

### CODE REVIEW

File: Directory.Build.targets [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <Project>
+  <Import Project="build\sources.props" />
```
Comment (natemcmaster) [inline Directory.Build.targets]: Don't merge back any of the project-system changes. That includes any changes to .props, .targets, .csproj, or NuGet.config files.

File: build.sh [hunk 2]
```diff
@@ -23,7 +23,7 @@ tools_source=''
 # Functions
 #
 __usage() {
-    echo "Usage: $(basename ${BASH_SOURCE[0]}) [options] [[--] <MSBUILD_ARG>...]"
```
Comment (natemcmaster) [inline build.sh]: Any build.sh/ps1/cmd scripts too

File: Directory.Build.targets (hunk 1 above)
Comment (smitpatel) [reply Directory.Build.targets]: Please add a commit to reverse all the changes which are not supposed to merged back in.

**Comment by smitpatel** [comment]
Thanks @AndriySvyryd for strategies.

**Pull request closed without merging**